apm-sniffer/apm-sdk-plugin/jdbc-commons/src/main/java/org/skywalking/apm/plugin/jdbc/ConnectionServiceMethodInterceptor.java

```diff
@@ -27,15 +27,11 @@
 import org.skywalking.apm.agent.core.plugin.interceptor.enhance.InstanceMethodsAroundInterceptor;
 import org.skywalking.apm.agent.core.plugin.interceptor.enhance.MethodInterceptResult;
 import org.skywalking.apm.plugin.jdbc.trace.ConnectionInfo;
-import org.skywalking.apm.util.StringUtil;
 
 /**
- * {@link ConnectionServiceMethodInterceptor} create an exit span when the client call the following methods in the class
- * that extend {@link java.sql.Connection}.
- * 1. close
- * 2. rollback
- * 3. releaseSavepoint
- * 4. commit
+ * {@link ConnectionServiceMethodInterceptor} create an exit span when the client call the following methods in the
+ * class that extend {@link java.sql.Connection}. 1. close 2. rollback 3. releaseSavepoint 4. commit
```

Comment:
* when the following methods execute
* No need of `that extend` words. Just `:`